tests/shared/src/test/scala/matryoshka/spec.scala

```diff
@@ -1018,27 +1018,29 @@ class MatryoshkaSpecs extends Specification with ScalaCheck with ScalazMatchers
   }
 
   "Attr" >> {
+    type T[A] = Cofree[Exp, A]
+
     "attrSelf" >> {
       "annotate all" >> Prop.forAll(expGen) { exp =>
-        exp.cata(attrSelf[Mu[Exp], Exp]).elgotPara(universe) must
-          equal(exp.elgotPara(universe).map(_.cata(attrSelf[Mu[Exp], Exp])))
+        exp.cata(attrSelf[T[Mu[Exp]]].apply).elgotPara(universe) must
```

Comment:
Meh. I think it’s unavoidable until we have scala/scala#5108, but I really hate this `.apply` stuff.